### DISCUSSION TITLE
Add more MPPTs in Solar Options

### DISCUSSION THREAD

**Discussion by cscherpenisse** [slipx06/sunsynk-power-flow-card#370] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

-

### Expected behaviour

Is it possible to see more MPPTs?
For now it is a maximum of 4, but can this be increased to 8 or higher?
I would like to display my solar panels individually.

### Possible Solutions

-

### Mode

Both

### Context / Reason

-

**Comment by slipx06** [comment]
Hi it has been requested before but due to the limited space not possible until I get around to designing a wider version of the card.

**Comment by slipx06** [comment]
Also discussed https://github.com/slipx06/sunsynk-power-flow-card/discussions/291. Adding this to ideas
